### ISSUE TITLE
Jenkins Plugin not available in public plugins

### ISSUE THREAD

**Issue by stevegal** [capitalone/Hygieia#1387]
The plugin should be pushed so that it's available in the normal jenkins plugin list via the update centre.

Are there any plans/timelines to publish this (it's not too difficult a process as I have already published a plugin there).

If there are no plans, would it be appropriate for me to publish this?

**Comment by rob-miller-777** [comment]
It would also be nice if it was a java 7 version of the plugin as well.

**Comment by tabladrum** [comment]
Yes. We have some code in core that is Java 8. Wish I have some time to clear that off.

**Comment by tabladrum** [comment]
@stevegal yes, that is our goal. I have gone through the process documentation too. Just need to spend one night and finish it off :-)

**Comment by rob-miller-777** [comment]
I created a branch that made core java 7, there was really only one class that was using java 8 if I remember correctly.  I could create a merge request for that if you guys were interested.

**Comment by stevegal** [comment]
@tabladrum cool. I suppose it'll give you a chance to include https://github.com/capitalone/Hygieia/pull/1368 in the first release ;-)

**Comment by tsutsarin-fuib** [comment]
@tabladrum Any plans on publishing to Jenkins Update Center?
